ui/src/views/Test.vue

```diff
@@ -76,11 +98,37 @@ export default {
         this.testLog = res
       })
     },
+    async runExport(directExport) {
+      if (directExport.value === 'export_image' && this.isdisabled === false) {
+        this.isdisabled = true
+      }
+      await axios
+        .get(`${getFileServerPath()}${this.workspace}${directExport.path}`, { responseType: 'blob', params: { is_export_stream: true, export_value: directExport.value } })
+        .then((response) => {
+          const blob = new Blob([response.data], { type: 'application/zip' })
+          const link = document.createElement('a')
+          link.href = URL.createObjectURL(blob)
+          link.download = `${directExport.name}.zip`
+          link.click()
+          URL.revokeObjectURL(link.href)
+        }).catch((error) => {
+          console.error(error)
+        })
+      if (directExport.value === 'export_image' && this.isdisabled === true) {
+        this.isdisabled = false
+      }
+    },
+    async checkFolderExist() {
+      const workspace = await api.getWorkspace()
+      const exportImage = await api.checkFolder(workspace)
+      this.doesFolderExist.export_image = exportImage === 'ok'
```

Comment:
You should set back to default value if it does not exist. The state in the backend can always change so that can be toggled then 
Where do you check for the other cases? Right now you simply check if the workspace exist which should be the case as we are in train view of the previously selected workspace. You should check (and set enabled state) for cases "export model" and "export results". So in case of model you can check if folder "model" exists in workspace, same goes for results. To be very clean here, checking for existence of results could be to  try to download the file by trying a GET /<someworkspace>/model/[test|train]_scores.csv. In case of huge files this will be a mess as client waits for download to be done which can take time (so resolving button state will be also pending). A very elegant solution would be to add another service into controller to offer a HEAD /<someworkspace>/model/[test|train]_scores.csv. The HTTP method HEAD in comparison to GET does not trigger the full download and return full content in response body, HEAD can be used to retrieve any metadata apart from the blob payload. So in our case we could add in router and respective FileController another service/method/function that is configured in the API route HEAD /anyfile and simply try to find "anyfile" in backend and if it is a file (not directory) and readable simply return the fs.Stats object in response body that is retrieved by fs.stats call used in your PR. If file not found or accessible simply return a 404 (not found), 401 (not accessible) on the HEAD request and a 200 together with stats object in case of success. That response can be used to query the files of interest (test_scores.csv and train_scores.csv) and link the outcome with the button enabled state.

The reason why I am so expressive here is that we could use later such a HEAD call for other use cases in the feature.